docs/topics/proc_using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+
+link:https://docs.openrewrite.org/[OpenRewrite], which is integrated into the {ProductShortName} CLI, uses recipes to automate large-scale, distributed source code refactoring.
+
+You can use OpenRewrite in the {ProductShortName} CLI to prepare Java applications for migration.
+
+[IMPORTANT]
+====
+OpenRewrite recipe support is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
+The `JavaxtoJakarta` recipe renames imported `javax` packages to their `jakarta` equivalents.
+
+.Procedure
+
+. Specify the OpenRewrite recipe when you run `mta-cli`:
++
+[source,terminal]
+----
+$ ./mta-cli --openrewrite "-DactiveRecipes=<recipe name>"  \
+  --input /path/to/source/project --goal dryRun
+----
+
+* `--openrewrite`: Flag specifying to run an OpenRewrite migration, instead of an MTA analysis.
+
+* `"-DactiveRecipes=<recipe name>"` : The OpenRewrite recipe to apply to the input project. `org.jboss.windup.JavaxtoJakarta` is the default shipped recipe but you can add your own to the shipped `rewrite.yml` and run that recipe instead. The `rewrite.yml file` is located in the `rules/openrewrite/`` folder in the uncompressed MTA distribution.
+
+* `--input`: The application to be evaluated.
+
+* `--goal`: Optional: The OpenRewrite Maven goal to run. Parameters:
+** `dryRun` : The script returns a list of proposed changes. By default, the output contains a message – `"Run 'mvn rewrite:run' to apply the recipes"` – that you should ignore.
```

Comment:
Why should the user ignore the message to run the command? That needs to be explained.

Are you using an en/em dash here? I am fairly certain that we are not supposed to use them. This is not good coding. If our users are supposed to ignore the message, our devs should be able to suppress the output but I'll let it go. Just do something about the punctuation.